intact-orthology-import/src/main/java/uk/ac/ebi/intact/ortholog/OrthologsFileParser.java

```diff
@@ -0,0 +1,49 @@
+package uk.ac.ebi.intact.ortholog;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class OrthologsFileParser {
+
+    private static final Logger log = Logger.getLogger(OrthologsFileParser.class.getName());
```

Comment:
You can add the annotation `@Log4j` to the class and then you don't need this static object.
```
...

import lombok.extern.log4j.Log4j;
...

@Log4j
public class OrthologsFileParser {
...
```